### DISCUSSION TITLE
1109 Meetup PPT

### DISCUSSION THREAD

**Discussion by luzizhuo** [OpenSPG/KAG#45] [Show and tell]
PPT materials for the Meetup on November 9th.

[KAG-知识增强⽣成的垂域知识库知识服务框架-桂正科.pdf](https://github.com/user-attachments/files/17741194/KAG-.-.pdf)
[MedSPG：高血压诊疗领域层次化知识建模与多步推理决策-周庚显.pdf](https://github.com/user-attachments/files/17741427/MedSPG.-.pdf)
[TuGraph-DB 5.0：面向AI时代的图存储能力增强-王志勇.pdf](https://github.com/user-attachments/files/17741490/TuGraph-DB.5.0.AI.-.pdf)
[GraphRAG在DB-GPT中的实践与咨询行业应用案例分享.pdf](https://github.com/user-attachments/files/17741492/GraphRAG.DB-GPT.pdf)

**Comment by huangheLee** [comment]
Thanks for sharing.

**Comment by jacarrichan** [comment]
多谢分享
